script.js

```diff
@@ -1,4 +1,338 @@
+    console.log(playerHand, computerHand);
+    if (getHandSum(playerHand) > twentyOne) {
+      gameOver = true;
+
+      outputDiv.classList.add("color-change-lose");
+
+      setTimeout(function () {
+        outputDiv.classList.remove("color-change-lose");
+      }, 3000);
+
+      return `${getDefaultOutput()}. Player has over 21 and lost<br />Please press submit again to restart the game<br /><br />${myImage}`;
+    } else if (getHandSum(computerHand) > twentyOne) {
+      gameOver = true;
+      outputDiv.classList.add("color-change-win");
+
+      setTimeout(function () {
+        outputDiv.classList.remove("color-change-win");
+      }, 3000);
+      return `${getDefaultOutput()}. You won! Computer has over 21 and lost<br />Please press submit again to restart the game<br /><br />${myImage2}`;
+    } else if (getHandSum(playerHand) < 16) {
+      return `${getDefaultOutput()} <br />Hint: Psst, you need to hit...<br />Enter hit to draw another card or enter stand to, well, just stand 😅`;
+    } else if (getHandSum(playerHand) == 16) {
+      return `${getDefaultOutput()} <br />Hint: Could go either way. Press hit if you're feeling lucky 😋<br />Enter hit to draw another card or enter stand to, well, just stand 😅`;
+    } else {
+      return `${getDefaultOutput()} <br />Hint: Might want to stand on this one 😳<br />Enter hit to draw another card or enter stand to, well, just stand 😅`;
+    }
+  }
+
+  if (isBlackjack(playerHand)) {
+    gameOver = true;
+    outputDiv.classList.add("color-change-win");
+
+    setTimeout(function () {
+      outputDiv.classList.remove("color-change-win");
+    }, 3000);
+    return `Player has blackjack and wins. Please press submit again to restart the game!<br /><br />${myImage2}`;
+  }
+
+  if (isBlackjack(computerHand)) {
+    gameOver = true;
+    outputDiv.classList.add("color-change-lose");
+
+    setTimeout(function () {
+      outputDiv.classList.remove("color-change-lose");
+    }, 3000);
+    return `Computer has blackjack and wins. Please press submit again to restart the game!<br /><br />${myImage}`;
+  }
+
+  if (!playerHasChosenToStand) {
+    if (input !== "hit" && input !== "stand") {
+      console.log(playerHasChosenToStand);
+      return `Please submit "hit" or "stand"`;
+    }
+
+    if (input === "hit") {
+      dealCardToHand(playerHand);
+      if (getHandSum(playerHand) > twentyOne) {
+        gameOver = true;
+        outputDiv.classList.add("color-change-lose");
+
+        setTimeout(function () {
+          outputDiv.classList.remove("color-change-lose");
+        }, 3000);
+        return `<br>${getDefaultOutput()}</br>Player has over 21 and lost. Please press submit again to restart the game! <br /><br />${myImage}`;
+      }
+    }
+
+    if (input === "stand") {
+      playerHasChosenToStand = true;
+    }
+  }
+
+  // computer's turn now
+  var computerCardRankSum = getHandSum(computerHand);
+  if (computerCardRankSum <= dealerHitThreshold) {
```

Comment:
hmm... this seems to only deal one card if the sum is less than 16. what happens if after dealing the card, it is still less than 16?
You need a while loop to keep dealing until the sum for the dealer is less than 16